cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -123,6 +123,19 @@ func RunApply(flags *applyFlags) error {
 	internalcfg := &kubeadmapi.MasterConfiguration{}
 	api.Scheme.Convert(upgradeVars.cfg, internalcfg, nil)
 
+	// Validate requested and validate actual version
+	if err := configutil.NormalizeKubernetsVersion(internalcfg); err != nil {
```

Comment:
NormalizeKubernetesVersion

Reply:
heh. typo :(
BTW, maybe make it as interface to MasterConfiguration ? so it will be
```go
if err := internalcfg.NormalizeKubernetesVersion(); err != nil {
...
```
instead of `configutil.NormalizeKubernetesVersion(internalcfg)` ? 
This function is anyway usable only with that type.

Reply:
haven't decided how to generally deal with that, so as-is is fine for now